src/pages/[slug].tsx

```diff
@@ -1,14 +1,25 @@
-import { GetStaticProps } from "next";
+import { GetStaticProps, GetStaticPaths } from "next";
 import Head from "next/head";
-import { position } from "polished";
-import PositionCard from "../components/modules/IdeaRecruitPage/PositionCard";
+import PositionCard from "../components/modules/DetailedPage/PositionCard";
 import { Project } from "../components/modules/MainPage/filtering/project";
 
+export const getProjectsSlugs = async () => {
+  const res = await fetch("https://api.devlaunchers.org/projects");
+  const result: Project[] = await res.json();
+  const projects = result.filter((p) => p.openPositions.length > 0);
+
+  const projectsSlugs = projects.map((project) => ({
+    params: {
+      slug: project.title,
+    },
+  }));
+
+  return projectsSlugs;
+};
+
 // Load the first project for testing purpose
-export const getStaticProps: GetStaticProps = async (context) => {
-  const res = await fetch(
-    "https://api.devlaunchers.org/projects/dev-launchers-platform"
-  );
+export const getStaticProps: GetStaticProps = async ({ params }) => {
+  const res = await fetch("https://api.devlaunchers.org/projects/dev-launchers-platform");
```

Comment:
why do we only fetch "dev-launchers-platform" this should be dynamic, params has a property of slug that can used to fetch that specific endpoint rather than "dev-launchers-platform"

something like 
```js
 const res = await fetch(`https://api.devlaunchers.org/projects/${params.slug}`);
 ```